src/Action/CreateCommentAction.php

```diff
@@ -60,20 +66,27 @@ final class CreateCommentAction extends Controller
      */
     private $mailer;
 
+    /**
+     * @var EventDispatcherInterface
+     */
+    private $eventDispatcher;
+
     public function __construct(
         RouterInterface $router,
         BlogInterface $blog,
         PostManagerInterface $postManager,
         CommentManagerInterface $commentManager,
         FormFactoryInterface $formFactory,
-        MailerInterface $mailer
+        MailerInterface $mailer,
+        EventDispatcherInterface $eventDispatcher
```

Comment:
Technically a BC-break, but this is an action, so maybe this is OK?

Reply:
Good question 🤔 

The action is final, so this should be okay, but you could decorate this 😞 

I'll make this optional for now

Reply:
Fixed